### ISSUE TITLE
Away status

### ISSUE THREAD

**Issue by SamuelMoraesF** [glowing-bear/glowing-bear#598]
Will be great an box(maybe at the right of the input bar) with the user's nickname and the away status.

![captura de tela de 2015-04-14 22 38 51](https://cloud.githubusercontent.com/assets/3705263/7150847/3a334cf8-e2f7-11e4-931a-53102b20f7f9.png)

**Comment by torhve** [comment]
If someone wants to submit a patch for that maybe we could include it.

**Comment by lorenzhs** [comment]
If we want to add this, it would be important not to show it on mobile, as space in the input bar is scarce.

**Comment by t3chguy** [comment]
Maybe on Mobile just show a small - single character symbol instead of (away)

**Comment by lorenzhs** [comment]
That would be unintuitive as it has absolutely no meaning if you don't already know what it means. UI is hard :(

**Comment by Maffblaster** [comment]
I like the idea of an away status for the web UI (as displayed in @SamuelMoraesF's image above).

**Comment by Bakke** [comment]
How about putting nickname and status in the top-left corner, with the Glowing Bear-logo and unread count?

**Comment by lorenzhs** [comment]
yeah that might work if it doesn't look to detached from the rest of the UI

**Comment by Techman-** [comment]
>That would be unintuitive as it has absolutely no meaning if you don't already know what it means. UI is hard :(

What you could do is make the "A" the same color as away nicks in weechat.
